scripts/ci-apidiff.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+cd "${REPO_ROOT}"
+
+APIDIFF="${PWD}/hack/tools/bin/go-apidiff"
+PULL_BASE_SHA=${PULL_BASE_SHA:-$(git rev-parse origin/master)}
```

Comment:
this is a best effort attempt to make the script work locally but the real use of this will be through the prow job which has `PULL_BASE_SHA` already set.